formal-spec/update-protocol.tex

```diff
@@ -3,96 +3,110 @@ \section{Update protocol}
 \subsection{Activation phase}
 \label{sec:activation-phase}
 
-Once the implementation of a proposal is approved, it enters the
-\emph{activation phase}.
-%
-There can be \emph{at most one proposal} in the activation phase at any time.
+The activation  phase is the phase in the lifecycle of a 
+software update where the changes take effect. It is the 
+phase where the participants of the network have to
+actually download and install the update and start using
+the upgraded ledger. This phase follows immediately after the Approval phase where the submitted implementations are
+reviewed by the experts (delegated by the community) and
+through a voting process reach a decision, whether a submitted implementation of a software update will proceed
+to activation, or will be rejected.
+
+A software update might affect the way the \emph{consensus}
+works. This is because it can change the rules defining
+what is a valid transaction, what is a valid block, what is a valid chain, or what chain to follow in the case of a
+fork. Moreover, it can change the underlying consensus protocol of the blockchain per se. For this reason, parties
+need to synchronize prior to the activation of such
+changes, in order to avoid a chain split, due to lack of consensus. Moreover, before launching the upgraded ledger,
+we need to make sure that the security assumptions hold, i.e., that enough honest stake has upgraded. To this end,
+the activation phase consists of a synchronization mechanism, which is called the \emph{endorsement period}
+that ensures that parties will activate changes synchronously and that the upgraded ledger will be secure.
+
+Approved implementations enter the activation phase based on their \emph{priority} and order in a priority queue that we call the \emph{activation queue}. We call the proposals in the activation queue the \emph{approved proposals}. The order of the approved proposals corresponds to the activation priority that we have assigned to them. This priority is metadata-driven and has been confirmed by the community during the Approval phase.
+
+The first approved proposal in the activation queue will enter the endorsement period. There can be \emph{at most one proposal} in the endorsement period at any time.
```

Comment:
A proposal can't be in a period. I'd say a proposal can be in the activation phase. The activation phase for a proposal consists of multiple endorsement periods.